src/services/users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { createNewId } = require('../helpers.js');
+
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+const getAll = () => users;
+const getById = (id) => {
```

Comment:
Let's add some break lines between functions

```suggestion
};

const getAll = () => users;

const getById = (id) => {
```